modules/verify-knative-serving-monitoring.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * serverless/monitoring-serverless.adoc
+
+[id="configuring-knative-services-domains_{context}"]
+= Verifying an {product-title} monitoring installation for use with Knative Serving
+
+The OpenShift Serverless operator deploys ServiceMonitor objects during the installation automatically.
```

Comment:
This seems to be mentioned in the doc right above this already.

Reply:
Removed :)